bridge/src/index.ts

```diff
@@ -378,6 +384,10 @@ process.on("uncaughtException", console.error);
         USE_NCSCAN_URL,
         ETHERSCAN_ROOT_URL,
         ncgExchangeFeeRatioPolicy,
+        {
+            criterion: BASE_FEE_CRITERION,
+            fee: BASE_FEE,
```

Comment:
A simple validity check (e.g., `BASE_FEE` should be less than `BASE_FEE_CRITERION`) might be helpful to prevent misconfiguration.

Reply:
Added validatio for `BASE_FEE`  and `BASE_FEE_CRITERION`

fixed to Throw Error when `BASE_FEE >= BASE_FEE_CRITERION`